src/nomad_material_processing/vapor_deposition.py

```diff
@@ -0,0 +1,417 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import (
+    TYPE_CHECKING,
+)
+from nomad.metainfo import (
+    Package,
+    Section,
+    SubSection,
+    Quantity,
+    MEnum,
+)
+from nomad.datamodel.data import (
+    ArchiveSection,
+)
+from nomad.datamodel.metainfo.annotations import (
+    ELNAnnotation,
+    ELNComponentEnum,
+)
+from nomad.datamodel.metainfo.basesections import (
+    ActivityStep,
+    CompositeSystem,
+    PureSubstanceSection,
+    ReadableIdentifiers,
+)
+from nomad_material_processing import (
+    SampleDeposition,
+    ThinFilmStackReference,
+    ThinFilmStack,
+    ThinFilmReference,
+    ThinFilm,
+)
+
+if TYPE_CHECKING:
+    from nomad.datamodel.datamodel import (
+        EntryArchive,
+    )
+    from structlog.stdlib import (
+        BoundLogger,
+    )
+
+m_package = Package(name="Vapor Deposition")
+
+
+class MaterialEvaporationRate(ArchiveSection):
+    m_def = Section(
+        a_plot=dict(
+            x="process_time",
+            y="rate",
+        ),
+    )
+    rate = Quantity(
+        type=float,
+        unit="mol/meter ** 2/second",
+        shape=["*"], ############ why is it an array? 
+        # a_eln=ELNAnnotation(
+        #     defaultDisplayUnit="micromol/m ** 2/second",
+        # ),
+    )
+    flow = Quantity(  ############### I need this, let's discuss if it must be here or in CVD module
+        type=float,
+        a_eln=ELNAnnotation(
+            defaultDisplayUnit="cm ** 3 / minute",
+        ),
+        unit="cm ** 3 / minute",
+    )
```

Comment:
we'll figure out better, but for now I place it in cvd

Reply:
why is `flow ` in `MaterialEvaporationRate`? But yes, (MO)CVD needs a `flow` class since you often times also have only carrier gas flows without any evapouraized material